weldx/tags/time/timedeltaindex.py

```diff
@@ -9,6 +9,14 @@
 __all__ = ["TimedeltaIndexConverter"]
 
 
+def _handle_converted_pd_tdi_units(node: TaggedDict):
```

Comment:
backward compat @CagtayFabry

Reply:
changelog of pandas:
> Units ‘H’, ‘T’, ‘S’, ‘L’, ‘U’ and ‘N’ are deprecated and will be removed in a future version. Please use ‘h’, ‘min’, ‘s’, ‘ms’, ‘us’, and ‘ns’ instead of ‘H’, ‘T’, ‘S’, ‘L’, ‘U’ and ‘N’.